packages/prometheus/0.3.4/manifest.yml

```diff
@@ -0,0 +1,33 @@
+format_version: 1.0.0
+name: prometheus
+title: Prometheus
+version: 0.3.4
+license: basic
+description: Prometheus Integration
+type: integration
+categories:
+  - monitoring
+  - datastore
+release: experimental
+conditions:
```

Comment:
@ChrsMark Have you tested this with 7.10.0 Kibana and Elastic Agent?

Reply:
It was not tested as part of the integrations' list of https://github.com/elastic/obs-dc-team/issues/513. Shall we abort promoting it to production because of this?

Reply:
As long as we did not test 7.10, 7.11 and 7.12, we should not promote this! Is there a reason we don't adjust the condition to only match 7.13 and beyond? I thougth that is our general rule for now to make sure we don't break old packages.

Reply:
Not promoting sounds good to me. I would avoid adjusting the condition for now since the specific package has not passed the migration assessment yet and this is why it was not part of the list at elastic/obs-dc-team#513. So let's postpone promoting it to production for now.